tests/integration/test_load_and_run_multi_agent.py

```diff
@@ -99,5 +99,12 @@ def test_load_and_run_multi_agent(
             assert cost_sum.total_cost < 1.00
             assert cost_sum.total_tokens > 0
             assert cost_sum.total_tokens < 20000
+        if agent_framework not in (AgentFramework.AGNO, AgentFramework.GOOGLE, AgentFramework.TINYAGENT):
```

Comment:
We now have a convenient `is_tracing_supported` function that you should use here.